src/perf/dbserver/CreatePlatform.ps1

```diff
@@ -0,0 +1,37 @@
+<#
+
+.SYNOPSIS
+This script creates a new platform on the DB Server
+
+.PARAMETER PlatformName
+    The Platform name to create
+
+.PARAMETER AuthKey
+    The Authorization Key to use
+
+#>
+
```

Comment:
nit: can we remove extra newlines (only have one) in these scripts?

Reply:
Done